scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -484,8 +484,10 @@ def device_is_available(self, instance):
         return None
 
     def make_dut_available(self, dut):
+        if self.instance.status in ["error", "failed"]:
```

Comment:
same as above

Reply:
to fix this problem twister needs to consider 'failed' test instances at a DUT:

> .. for instance to resolve ploblems when some DUTs have connectivity or HW issues slowing down test plan execution, or even block the execution when only one test suite runs whereas the same first DUT candidate in the list is not working and others were not chosen.